content/en/build-guide/m12.adoc

```diff
@@ -23,7 +23,11 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC
 
 == Assembly Diagram == 
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+
+== Circuit Diagram == 
+
```

Comment:
There's a previous/next button being generated above the PCBA drawing that doesn't seem to do anything (see next comment)

Reply:
Should be fixed